src/tests/memchr/mod.rs

```diff
@@ -305,3 +305,11 @@ impl Seed {
         more.into_iter()
     }
 }
+
+fn sync_regression() {
+    use crate::{Memchr, Memchr2, Memchr3};
+    fn assert_send_sync<T: Send + Send>() {}
```

Comment:
This should be `Send + Sync`. And also include `UnwindSafe` and `RefUnwindSafe`, as those are two other auto traits I've accidentally dropped in the past in other crates.

Also should be moved to the test section of `src/memchr.rs` and marked with `#[test]`.

Reply:
> This should be Send + Sync

Ah, oops!